api-reference/10 UI Widgets/dxDiagram/1 Configuration/nodes/autoLayout/type.md

```diff
@@ -8,4 +8,7 @@ type: String
 Specifies the diagram layout type.
 
 ---
-When the **type** option is set to **layered** or **tree** the shape coordinates specified by the [leftExpr](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#leftExpr) and [topExpr](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#topExpr) options are ignored.
\ No newline at end of file
+ The **auto** value corresponds to **off** when the both shape coordinates (<xref:DevExpress.Web.ASPxDiagram.DiagramNodeMappingInfo.Left> and <xref:DevExpress.Web.ASPxDiagram.DiagramNodeMappingInfo.Top>) are specified; otherwise, the **auto** value corresponds to **layered**.
```

Comment:
```suggestion
 The **auto** value corresponds to **off** when both shape coordinates (<xref:DevExpress.Web.ASPxDiagram.DiagramNodeMappingInfo.Left> and <xref:DevExpress.Web.ASPxDiagram.DiagramNodeMappingInfo.Top>) are specified; otherwise, the **auto** value corresponds to **layered**.
```